src/App.js

```diff
@@ -5,6 +5,13 @@ import "./App.css";
 
 // Logo has already been provided for you. Do the same for the remaining components
 import Logo from "./components/DisplayComponents/Logo";
+import Display from "./components/DisplayComponents/Display";
+import Numbers from "./components/ButtonComponents/NumberButtons/Numbers";
+import NumberButton from "./components/ButtonComponents/NumberButtons/NumberButton";
+import Operators from "./components/ButtonComponents/OperatorButtons/Operators";
+import OperatorButton from "./components/ButtonComponents/OperatorButtons/OperatorButton";
+import Specials from "./components/ButtonComponents/SpecialButtons/Specials";
+import SpecialButton from "./components/ButtonComponents/SpecialButtons/SpecialButton";
```

Comment:
Since your importing NumberButton, SpecialButton, and OperatorButton into their parent components, you don't need to import them into App.js.